transformation_matrix:
- 0.41469324265623914
- -0.9099612708765432
- 0.0
- 0.0
- 0.9099612708765432
- 0.41469324265623914
- 0.0
- 0.0
- 0.0
- 0.0
- 1.0
- 0.0
- 0.0
- 0.0
- 0.0
- 1.0
yaw_angle_degrees: 65.5
